### ISSUE TITLE
Binoculars emoji

### ISSUE THREAD

**Issue by Crissov** [Crissov/unicode-proposals#125]
Binoculars often symbolize finding/searching, looking for something or spying on someone from a distance. Pirates used “monoculars” and there’s a Monocle emoji coming in Unicode 10.

**Comment by accordionnoir** [comment]
Did you submit a binoculars emoji proposal? I see that Unicode declined one in 2021. ☹️ Some birder friends were thinking about resubmitting, but it would be nice to see it any older proposals 🙏🏼

**Comment by Crissov** [comment]
@accordionnoir No, that was someone else. Emojination lists Binoculars as a suggestion without anyone working on it.
